### PR TITLE
Added build_script,Dockerfile,build_info file for rethinkdb

### PR DESCRIPTION
**Pull request by vinodk99**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

**Comment by vinodk99** [comment]
Travis check is failing due to invalid argument ,Tried to build and test the script in UBI_8 container and attached below logs.
[RethinkDB_Script_logs.txt](https://github.com/ppc64le/build-scripts/files/10256451/RethinkDB_Script_logs.txt)

**Comment by seth-priya** [comment]
@nikhil-kalbande please review

File: r/rethinkdb/rethinkdb_ubi_8.5.sh [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+
+PACKAGE_VERSION=v2.4.x
```
Comment (nikhil-kalbande) [inline r/rethinkdb/rethinkdb_ubi_8.5.sh:22]: @vinodk99 - Can you make change to accept dynamic PACKAGE_VERSION  value with $1 parameter to script `PACKAGE_VERSION=${1:-v2.4.x}`

File: r/rethinkdb/rethinkdb_ubi_8.5.sh [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+ln -s /usr/bin/python2 /usr/bin/python
+
+git clone $PACKAGE_URL
+cd $PACKAGE_NAME
+
+git checkout $PACKAGE_VERSION
+
+./configure --allow-fetch
+make
```
Comment (nikhil-kalbande) [inline r/rethinkdb/rethinkdb_ubi_8.5.sh:47]: @vinodk99 please capture build/test success and failures with return codes.

**Review by nikhil-kalbande — CHANGES_REQUESTED** [review]

File: r/rethinkdb/rethinkdb_ubi_8.5.sh (hunk 2 above)
Comment (vinodk99) [reply r/rethinkdb/rethinkdb_ubi_8.5.sh]: @nikhil-kalbande added build and test return codes to the script, and also added build and test logs which run in the power vm.
[RethinkDB_Build_and_test_logs.txt](https://github.com/ppc64le/build-scripts/files/10542975/RethinkDB_Build_and_test_logs.txt)

**Review by nikhil-kalbande — APPROVED** [review]

**Pull request merged**